src/arm.c

```diff
@@ -22,6 +22,7 @@
 /* opcode */
 typedef enum {
     arm_and = 0,
+    arm_xor = 1,
```

Comment:
The instruction is called `EOR` rather than `XOR`.
See https://developer.arm.com/documentation/dui0489/h/arm-and-thumb-instructions/and--orr--eor--bic--and-orn

Reply:
Actually, I use `EOR` unstruction. `XOR` is just for its naming convention. Should I change them to `eor` for naming?

Reply:
You should use `EOR` for consistency with Arm reference manual.